vlrouter.go

```diff
@@ -236,6 +248,14 @@ func (self *Vlrouter) RemoveLocalEndpoint(endpoint OfnetEndpoint) error {
 		log.Errorf("Error deleting the endpoint: %+v. Err: %v", endpoint, err)
 	}
 
+	// Remove the endpoint from policy tables
+	if endpoint.EndpointType != "internal-bgp" {
```

Comment:
loopback interface endpoint type is internal-bgp and its installed as a local endpoint.